provider/caleta/api_client.go

```diff
@@ -133,11 +134,19 @@ func (apiClient *apiClient) getGameRoundRender(ctx context.Context, gameRoundID
 
 }
 
+type transactionRequestBody struct {
```

Comment:
In the future, maybe move this to api_model.go?

Reply:
Ah of course! Fixed